vms/pom/pages/adminRegistrationPage.py

```diff
@@ -28,9 +28,9 @@ def fill_registration_form(self, info):
         self.send_value_to_element_id(elements.LAST_NAME, info[4])
         self.send_value_to_element_id(elements.EMAIL, info[5])
         self.send_value_to_element_id(elements.ADDRESS, info[6])
-        self.send_value_to_element_id(elements.CITY, info[7])
-        self.send_value_to_element_id(elements.STATE, info[8])
         self.send_value_to_element_id(elements.COUNTRY, info[9])
+        self.send_value_to_element_id(elements.STATE, info[8])
```

Comment:
Why this change?

Reply:
To resemble that the order is chnaged in which we fill the form.